modules/openfast-library/src/FAST_Solver.f90

```diff
@@ -513,7 +517,12 @@ SUBROUTINE AD_InputSolve_NoIfW( p_FAST, u_AD, y_ED, BD, MeshMapData, ErrStat, Er
       
             
    END IF
-   
+        ! Set Conrol parameter (i.e. flaps)
+      DO k_bl=1,p_AD%numBlades
+          DO k_bn=1,p_AD%NumBlNds
+              u_AD%UserProp(k_bn , k_bl) = y_SrvD%BlFlapCom(k_bl) !bem: This takes in flap deflection for each blade (only one flap deflection angle per blade) from ServoDyn (which comes from Bladed style DLL controller)
```

Comment:
I'd personally prefer not to use the `p_AD` data type in the glue code unless absolutely necessary. In this case, you could loop from `k=1 to size(u_AD%UserProp,2)` and set `u_AD%UserProp(:,k) = y_SrvD%BlFlapCom(k)`

Then you don't need to pass the AD parameter type to this routine at all.

Reply:
Also, you should put an `if (p_FAST%CompServo == Module_SrvD` around this so you don't get seg faults if you use AeroDyn but not ServoDyn.

Reply:
Good catch!  I've added this as well.